.github/ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Chore
+about: Suggest a chore that will help contributors and doesn't affect end users.
```

Comment:
```suggestion
about: Suggest a chore that will help contributors and doesn't affect end users
```

Reply:
(I don't know... the other one didn't have a period)

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/581/commits/a15705e2af10bfc1bb15cbc4a29d592b8e4cc063.